maro/simulator/scenarios/finance/README.md

```diff
@@ -0,0 +1,149 @@
+# Finance scenario in MARO
+
+## Introduction
+
+The Finance scenario simulates securities trading in financial markets.
+Imagine you invest in a finance market. For example, CN Shanghai:
+You can create a strategy which decides how to buy and sell stocks.
```

Comment:
We can develop a strategy for buying and selling stocks, which try to maximize revenue.

Reply:
ok